web/src/pm_list.ts

```diff
@@ -39,8 +39,10 @@ export function close(): void {
 }
 
 export function _build_direct_messages_list(): vdom.Tag<PMNode> {
-    const conversations = pm_list_data.get_conversations();
-    const pm_list_info = pm_list_data.get_list_info(zoomed);
+    const $filter = $<HTMLInputElement>(".direct-messages-list-filter").expectOne();
+    const search_term = $filter.val() ?? "";
```

Comment:
I don't think `?? ""` is correct to write here. The value will be a string. I think a type assertion could be appropriate, like `search_term = $filter.val()!`, if you can't teach TypeScript that it's a string, which we do pretty often. But using `??` incorrectly suggests that there's a conditional possibility here, and we should not do that, because it makes code harder to read that you're now thinking about whether `""` is correct if this were to be somehow undefined (which it probably would be, but still, the point is to avoid implying there are branches that don't exist).